src/main/java/com/duke/tasks/Todo.java

```diff
@@ -9,7 +9,7 @@ public Todo(String description) {
     @Override
     public String getSaveDescription() {
         return String.format("%s | %s | %s ",
-                getClass().getName(), status == true ? 1 : 0, description);
+                getClass().getSimpleName(), status == true ? 1 : 0, description);

```

Comment:
I like that getSimpleName goes into more detail of the name, just curious is there a more detailed name you wish to return as well?

Reply:
The getName() method worked for me fine until I packaged my classes, which following that it would give me:

"com.duke.task.Todo" 

instead of 

"Todo" 

So I had to change it into getSimpleName() instead,